tests/phpunit/ExceptionConverterTest.php

```diff
@@ -15,24 +17,16 @@ class ExceptionConverterTest extends TestCase
     /**
      * @dataProvider exceptionMessageProvider
      */
-    public function testHandleCopyFilesExpectedUserException(string $message): void
-    {
-        $this->expectException(UserException::class);
+    public function testHandleCopyFilesException(
+        string $expectedException,
+        string $message,
+        string $throwException
+    ): void {
+        $this->expectException($expectedException);
         $this->expectExceptionMessage($message);
 
         try {
-            throw new \RuntimeException($message);
-        } catch (\Throwable $e) {
-            ExceptionConverter::handleCopyFilesException($e);
-        }
-    }
-
-    public function testHandleCopyFilesExpectedApplicationException(): void
-    {
-        $this->expectException(ApplicationException::class);
-
-        try {
-            throw new \RuntimeException('Foo bar');
+            throw new $throwException($message);
```

Comment:
čekal jsem teda, že tady už nebudeš dělat new - někdy můžeš chtít otestovat jinou message, než kterou vyhazuješ - např.  u `FileNotFoundException` je jiná message. Tak proto jsem čekal `string $expectedExceptionMessage` a `\Throwable $thrownException`...

Reply:
Jo tak to bude lepší, upraveno